_data/changelogs/accessibility.yml

```diff
@@ -1,9 +1,8 @@
-title: Accessibility
-pageURL: accessibility
+pageName: Accessibility
+pageType: Guidance
 items:
   - date: 2021-05-20
-    description:
-      summary: Updated all content to better explain current standards.
-      additional: This update explained USWDS accessibility standards, provided guidance on how to reach accessibility compliance on projects, and provided links to helpful resources.
+    summary: Updated all content to better explain current standards.
+    moreSummary: This update explained USWDS accessibility standards, provided guidance on how to reach accessibility compliance on projects, and provided links to helpful resources.
```

Comment:
I feel like we were close before, but just needed small tweaks.

```yml
description: Updated all content to better explain current standards.
additional_info: |-
  This update explained USWDS accessibility standards, provided guidance on how to reach accessibility compliance on projects, and provided links to helpful resources.
```

Reply:
This is better. Thanks! Updated in [1d1a671](https://github.com/uswds/uswds-site/pull/1789/commits/1d1a67158cdfc137443d03ddb254ee6a90b061e3)